src/sidebars/version-directive.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.version')
+        .value('version', '0.1.0')
```

Comment:
I'd say that grunt should increment this. Devs will no doubt forget at least once.